build.gradle

```diff
@@ -57,9 +57,9 @@ buildscript {
     }
 
     dependencies {
+        classpath "org.jetbrains.kotlinx:atomicfu-gradle-plugin:$atomicfu_version"
```

Comment:
You should add a comment that this line is exactly here for a reason. Otherwise, I promise to forget about this PR when I start reordering all dependencies in the reverse alphabetical order for fun.